.github/workflows/release.yml

```diff
@@ -128,6 +133,15 @@ jobs:
             aarch64-linux-gnu-strip \
             /target/${{ matrix.target }}/release/hx
 
+      - name: Strip release binary (riscv64)
+        if: matrix.build == 'riscv64-linux'
+        run: |
+          docker run --rm -v \
+            "$PWD/target:/target:Z" \
+            rustembedded/cross:${{ matrix.target }} \
+            riscv64-linux-gnu-strip \
+            /target/${{ matrix.target }}/release/hx
```

Comment:
I think this could be combined with the above one, right? It would just need to replace the target with `${{ matrix.target }}-gnu-strip`

Reply:
Rust target names and the GCC target names selected by cross-rs don't quite match up. E.g.:

| Rust | GCC from cross-rs image |
| --- | --- |
| `aarch64-unknown-linux-gnu` | `aarch64-linux-gnu` |
| `riscv64gc-unknown-linux-gnu` | `riscv64-linux-gnu` |
| `armv7-unknown-linux-gnueabihf` | `arm-linux-gnueabihf` |

Reply:
Maybe we could do #3327 in a CI step so that the change only affects the release CI run.

Reply:
#3780